### PR TITLE
Update AngusJohnson.ResourceHacker.installer.yaml

### PR DESCRIPTION
**Pull request by SreakMan**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/161845)

### CODE REVIEW

File: manifests/a/AngusJohnson/ResourceHacker/5.2.6/AngusJohnson.ResourceHacker.installer.yaml [hunk 1]
```diff
@@ -21,8 +21,8 @@ FileExtensions:
 - res
 - scr
 AppsAndFeaturesEntries:
-- DisplayName: Resource Hacker Version 5.2.3
-  DisplayVersion: ''
+- DisplayName: Resource Hacker Version 5.2.6
```
Comment (ItzLevvie) [inline manifests/a/AngusJohnson/ResourceHacker/5.2.6/AngusJohnson.ResourceHacker.installer.yaml:24]:
```suggestion
- DisplayName: Resource Hacker version 5.2.6
```

Comment (ItzLevvie) [reply manifests/a/AngusJohnson/ResourceHacker/5.2.6/AngusJohnson.ResourceHacker.installer.yaml:24]: to be more consistent with Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Pull request merged**